cmd/dlt/autoscaler/cmd.go

```diff
@@ -52,6 +52,10 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if !confirm.Prompt(true, "Are you sure you want to delete the cluster autoscaler?") {
+		os.Exit(0)
```

Comment:
I think that's just how we currently do it in general
https://github.com/openshift/rosa/blob/e97f178cc4c0ea96a64a8766a9ed995e5c9a7850/cmd/dlt/cluster/cmd.go#L87

There's also a confirm.Confirm instead of having to write "Are you sure ..."